ql/src/java/org/apache/hadoop/hive/ql/io/AcidUtils.java

```diff
@@ -230,6 +230,20 @@ public boolean accept(Path path) {
     }
   };
 
+  public enum CompactionColumn {
```

Comment:
redundant, just add cc_ to the column name.

Reply:
Added UI input->Schema mapping enum to ShowCompactionsAnalyzer. We cannot take name as it is which is in output. It will be like this
COMPACTIONID("CC_ID"),
    PART_NAME("CC_PARTITION"),
    DB_NAME("CC_DATABASE"),
    TB_NAME("CC_TABLE"),
    TYPE("CC_TYPE"),
    STATE("CC_STATE"),
    WORKERHOST("CC_WORKER_ID"),
    ENQUEUETIME("CC_ENQUEUE_TIME"),
    POOLNAME("CC_POOL_NAME"),
    Worker("CC_WORKER_VERSION"),
    START_TIME("CC_START"),
    TXNID ("CC_TXN_ID"),
    HADOOPJOBID("CC_HADOOP_JOB_ID");

This has to go in hive show compaction wiki